io-engine/src/lvs/lvs_lvol.rs

```diff
@@ -637,11 +637,19 @@ impl LogicalVolume for Lvol {
         unsafe { spdk_blob_is_read_only(self.blob_checked()) }
     }
 
-    /// Return the size of the Snapshot in bytes.
+    /// Return the size of the Logical Volume in bytes.
     fn size(&self) -> u64 {
         self.as_bdev().size_in_bytes()
     }
 
+    /// Return the committed size of the Logical Volume in bytes.
+    fn committed(&self) -> u64 {
```

Comment:
The way control-plane treats committed, is the amount that the pool usage can grow to, rather than how much more it can grow.
We could have a separate field of how much more we can grow, though we probably currently get this from pool.commited-pool.used